### DISCUSSION TITLE
Disable Scroll on chat

### DISCUSSION THREAD

**Discussion by saikti** [runelite/runelite#16589] [Ideas]
Would it be possible to make a plugin or setting that ignores scrollwheel on chatbox in resizable mode and just zoom the client. Its pretty annoying when you are trying to zoom out but you scroll the chat instead

**Comment by cupcakesssss** [comment]
i like this idea, and understand the frustration

**Comment by metzzger** [comment]
I've been looking for this. Someone made it! (Thanks [mcLovin1981](https://runelite.net/plugin-hub/mcLovin1981)!)
[https://runelite.net/plugin-hub/show/chatbox-scroll-to-zoom](https://runelite.net/plugin-hub/show/chatbox-scroll-to-zoom)
